operator.yaml

```diff
@@ -688,6 +691,9 @@ spec:
               cluster:
                 description: Cluster what kind of cluster you're running (ie, plain
                   Kubernetes or OpenShift)
+                enum:
+                - Kubernetes
+                - OpenShift
```

Comment:
Is this in the status or spec? We should not expose it for users to change this value.

Reply:
It is in the latform CRD spec. Why don't we want to expose it?

Let's suppose we are on OpenShift but on a specific namespace the user wouldn't like to use some advanced features related to OpenShift because we are developing for a different target cluster, in this way we will let the users to be free to do that.

Reply:
IDK, this use case seems really weird to me

Reply:
We can add a validation check that on a non OpenShift cluster that behaviour won't be permitted. 
Anyway I'll track this in a separate task because it is not related to this activity.